corehq/ex-submodules/pillowtop/checkpoints/manager.py

```diff
@@ -49,7 +49,7 @@ def get_or_create_wrapped(self, verify_unchanged=False):
             ))
 
         self._last_checkpoint = checkpoint
-        return result
+        return checkpoint
```

Comment:
would it make sense to make this the default from `get_or_create_checkpoint` too and remove `DocGetOrCreateResult` entirely? didn't look at the other code that uses this so not sure if that's being used elsewhere